x-pack/plugins/session_view/public/components/detail_panel_host_tab/index.tsx

```diff
@@ -95,104 +114,291 @@ export const DetailPanelHostTab = ({ processHost }: DetailPanelHostTabDeps) => {
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`container.id: "${containerData.id}"`}
+                    tooltipContent={containerData.id}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {containerData.id}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>name</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`container.name: "${containerData.name}"`}
+                    tooltipContent={containerData.name}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {containerData.name}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>image.name</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`container.image.name: "${containerData.image.name}"`}
+                    tooltipContent={containerData.image.name}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {containerData.image.name}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>image.tag</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`container.image.tag: "${containerData.image.tag}"`}
+                    tooltipContent={containerData.image.tag}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {containerData.image.tag}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+              {
+                title: <DetailPanelListItem>image.hash.all</DetailPanelListItem>,
+                description: (
+                  <DetailPanelCopy
+                    textToCopy={`container.image.hash.all: "${containerData.image.hash.all}"`}
+                    tooltipContent={containerData.image.hash.all}
+                  >
+                    <EuiTextColor color="subdued" css={styles.descriptionSemibold}>
+                      {containerData.image.hash.all}
+                    </EuiTextColor>
+                  </DetailPanelCopy>
+                ),
+              },
+            ]}
+          />
+        </>
+      )}
+      {processOrchestrator && (
+        <>
+          <DetailPanelAccordion
+            id="metadataOrchestrator"
+            title="Orchestrator"
```

Comment:
localize

Reply:
actually, since this title is derived from the fieldset name, perhaps localization doesn't make sense for this case.